YahtzeeSim/Hand.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace YahtzeeSim
+{
+    public class Hand
+    {
+        public static List<int> currentDice = new List<int> { };

```

Comment:
```suggestion
        public static var currentDice = new List<int> ();
```

I mean, it doesn't matter, but intellij will suggest not specifying the type because it makes it easier to refactor. and since you're not adding values here, you don't need to initialize with { } (I _think_  this is an under the hood optimization but not 100% sure)